react/StoreLink.tsx

```diff
@@ -106,4 +115,4 @@ StoreLink.schema = {
   title: 'Link',
 }
 
-export default StoreLink
+export default injectIntl(StoreLink)
```

Comment:
Use `useIntl` hook instead

Reply:
Nice! I've done like that copying other apps and seeing that the `react-intl` version was a bit old.

Fixed